sacred/experiment.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         interactive: bool = False,
         base_dir: Optional[PathType] = None,
         additional_host_info: List[HostInfoGetter] = None,
+        save_git_commit: bool = True,
```

Comment:
Hmm, `save_git_commit` sounds like we store only the commit, but we actually do more than that. How about `capture_git` or `git_integration` instead?

Reply:
Maybe save_git_info?